### ISSUE TITLE
Can your algorithm be used in a non-image related problem?

### ISSUE THREAD

**Issue by san89** [orobix/Prototypical-Networks-for-Few-shot-Learning-PyTorch#17]
Dear author(s),

Can this algorithm be used in another context different than image classification?

Thank you in advance for your help,

Best regards,

**Comment by dnlcrl** [comment]
Sure, but you should at least adapt the model and change the dataset class to your needs. The algorithms works with features in output of a model, so it doesn't depend on the input as long as you use the same loss function. Please note that we ar enot the paper's authors, but you can find them here: https://github.com/jakesnell/prototypical-networks
